packages/sdk/src/index.ts

```diff
@@ -0,0 +1,24 @@
+import { Job } from './job';
+import { Pipeline } from './pipeline';
+export { JobStatus } from './utils';
+export { JobModel, PipelineModel, JobRunOption, PipelineInstallOption } from './interface';
+
+/**
+ * Pipcook daemon sdk for javascript
+ * the operations of pipeline and job is available for now.
+ */
+export class PipcookClient {
+  pipeline: Pipeline;
+  job: Job;
+
+  /**
+   * the constructor for PipcookClient
+   * @param host the daemon host, like 'http://192.168.1.50'
+   * @param port the port
+   */
+  constructor(host: string, port = 6927) {
```

Comment:
The type for port is needed.

Reply:
It's not needed if we offer a default value.

Reply:
Look at:
https://www.typescriptlang.org/docs/handbook/functions.html#optional-and-default-parameters